CONTRIBUTING.md

```diff
@@ -67,6 +67,15 @@ which uses [google-java-format](https://github.com/google/google-java-format) li
 * By default, all arguments/members are treated as non-null. Every argument/member that can be `null` must be annotated with `@Nullable`.
 * The project aims to provide a consistent experience across all the public APIs. It is important to ensure consistency (same look and feel) across different public packages.
 * Use `final` for public classes everywhere it is possible, this ensures that these classes cannot be extended when the API does not intend to offer that functionality.
+* In general, we use the following ordering of class members:
+    * static fields
```

Comment:
Oh - also should we call out final before non-final?

Reply:
yeah, that's probably a good call. I'll update.

Reply:
done!